src/utils/keyboardUtils.ts

```diff
@@ -70,7 +70,7 @@ export function onEditorNavigation({ key, target }: React.KeyboardEvent<HTMLDivE
       target instanceof HTMLSelectElement)
   ) {
     return target.matches(
-      '.rdg-editor-container > :only-child, .rdg-editor-container > label:only-child > :only-child'
+      '.rdg-editor-container > :only-child, .rdg-editor-container > label:only-child > :only-child, .rdg-editor-container div:only-child > label:only-child > :only-child'
```

Comment:
The selector is too broad otherwise
```suggestion
      '.rdg-editor-container > :only-child, .rdg-editor-container > label:only-child > :only-child, .rdg-editor-container > div:only-child > label:only-child > :only-child'
```

Reply:
Ah missed it. Thanks!